stable/redis/README.md

```diff
@@ -37,6 +37,20 @@ The command deploys Redis on the Kubernetes cluster in the default configuration
 
 > **Tip**: List all releases using `helm list`
 
+## Using password file
```

Comment:
IMHO we should include this section inside the `## Configuration` section, adding another level (`###`) and moving it to the below section